app/src/components/domain/RiskImminentEvents/Pdc/EventDetails/index.tsx

```diff
@@ -54,6 +86,84 @@ function EventDetails(props: Props) {
         hospital?: Exposure | null;
     } | null;
 
+    const stormPoints = useMemo(
+        () => {
+            if (isNotDefined(exposure)) {
+                return undefined;
+            }
+
+            const { storm_position_geojson } = exposure;
+
+            const stormPositions = (storm_position_geojson as unknown as unknown[] | undefined)
+                ?.filter(isValidPointFeature);
+
+            const stormGeoJson: GeoJSON.FeatureCollection<GeoJSON.Geometry> = {
+                type: 'FeatureCollection' as const,
+                features: [
+                    ...stormPositions?.map(
+                        (pointFeature) => ({
+                            ...pointFeature,
+                            properties: {
+                                ...pointFeature.properties,
+                            },
+                        }),
+                    ) ?? [],
+                ].filter(isDefined),
+            };
```

Comment:
Any reason we are creating this stormGeoJson object here?

Reply:
The calculation done here is for the Bar chart each chart axis value depends on the individual properties. Therefore, calculating from `storm_position_geojson`